.github/workflows/ci.yml

```diff
@@ -1,59 +1,65 @@
 name: CI
 
 on:
-    pull_request:
-    push:
-        branches:
-            - master
-            - develop
-            - feature/[a-z]+-[0-9]+/dev-*
-    workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+      - develop
+      - feature/[a-z]+-[0-9]+/dev-*
+  workflow_dispatch:
 
 jobs:
-    validation:
-        name: "CS, PHPStan, Security"
-        runs-on: ubuntu-18.04
-        strategy:
-            fail-fast: false
-            matrix:
-                php-version: [
-                    '7.4',
-                    '8.1'
-                ]
-
-        steps:
-            - uses: actions/checkout@v3
-
-            - name: Setup PHP
-              uses: shivammathur/setup-php@v2
-              with:
-                  php-version: ${{ matrix.php-version }}
-                  extensions: mbstring, ctype, iconv
-                  tools: composer:v2
-
-            - name: Composer get cache directory
-              id: composer-cache
-              run: |
-                  echo "::set-output name=dir::$(composer config cache-files-dir)"
-            - name: Composer cache
-              uses: actions/cache@v2
-              with:
-                  path: ${{ steps.composer-cache.outputs.dir }}
-                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
-                  restore-keys: |
-                      ${{ runner.os }}-composer-
-            - name: Composer validate
-              run: composer validate
-
-            - name: Composer version
-              run: composer --version
-
-            - name: Composer install
-              run: composer install
-
-            - name: Run PHPStan
-              run: composer stan
-
-            - name: Run CodeStyle checks
-              run: composer cs-check
+  validation:
+    name: "CS, PHPStan, Security"
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - '7.4'
+          - '8.0'
+          - '8.1'
+        dependencies:
+          - 'lowest'
```

Comment:
Please see https://spryker.atlassian.net/wiki/spaces/CORE/pages/2868248862/Standalone+module+testing#Setup-Github-Actions for correct approach

Reply:
To summarize: 
- 7.4+8.1 normal (no in between needed)
- Additional 7.4 lowest to check for min dep

Nothing else needed

Please also add a few smoke tests to make the result more meaningful and missing .editorconfig file
All files must have 4 space indentation as per our rules.

Reply:
build fails on php7.4 during composer install because we use "^5.0 || ^6.0" for the most of symfony packages. And during regular composer install it fails because the minimal version for Symfony packaged  ^6.0 is 8.0. So we can't run 7.4 as a part of a "normal" job. That is why we split 7.4 and 8.0

Reply:
U need to adjust your packages to work with it. 
That is the whole point of all those 3

Reply:
agree. As I understood the idea, in the beginning, was to use only 8 php version and run sdk under docker, but then 7.4 support was added due to integration issues. I'll update all dependencies to make it more compatible.
Thank you!

Reply:
We always have range min max here
Should always have been 7.4+ ^^

We will adjust this once We remove a Version in the future